templates/books-index.html

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+  <div ng-repeat="book in booksIndexCtrl.books">
+
+    <div class="row">
+      <div class="col-md-3 col-xs-12 thumbnail">
+        <img ng-src="{{book.image}}" alt="book image">
+      </div>
+
+      <div class="col-md-9 col-xs-12">
+        <ul class="list-group form-inline">
+
+          <li class="list-group-item">
+            <a ng-href='books/{{book._id}}'>
+              <h4 class='inline-header'>Book Title:</h4>
+              <span ng-hide="editing">{{book.title}}</span>
```

Comment:
Since you're not using "editing" on your index page, all of this is unnecessary.